### DISCUSSION TITLE
Map status codes to text

### DISCUSSION THREAD

**Discussion by frankz217** [fredlcore/BSB-LAN#613] [General]
I am using mqtt to feed the status of my heatpump (and other values) to home assitant. 

Parameter 8006 gives me an interger for a certain state which I translate towards understandable status message. This is kind of trial and error because every time a number show up in stead of the translations, it is a new status which I do not have translated yet. 

I have a hard time mapping the numbers towards the readable state. For example In my case '51' is Heatpump off and '125' is 'Deforst' 

I do see all the messages in the language files, but they do not map with my status code integers. 

I am using a Atlantic Excellia. Would be great if I could get a list and change my code accordingly.

**Comment by fredlcore** [comment]
The status numbers should be listed in your manual, at least to some extent. Some dumps that users have sent me, contained one, however, It is not clear whether all numbers have the same meaning, that's why we don't provide one in order not to generate a sense of false security.

**Comment by frankz217** [comment]
Thanks. Makes sense, I do not have any documentation about status. I do have the handbook which talks about config mostly, no mapping on messages. Will contact the vendor. Thanks!

**Comment by fredlcore** [comment]
Yes, that's the documentation I mean. Depending on the manufacturer, they include some or more of these messages, but the safest way is to contact the manufacturer (and hope they are giving out this kind of information)...
